src/arch/xtensa/configs/imx8_defconfig

```diff
@@ -2,3 +2,4 @@ CONFIG_IMX8=y
 CONFIG_HAVE_AGENT=n
 CONFIG_FORMAT_CONVERT_HIFI3=n
 CONFIG_KPB_FORCE_COPY_TYPE_NORMAL=n
+CONFIG_SAMPLE_KEYPHRASE=y
```

Comment:
@cristinailie I don't think adding an option to defconfig is needed because once you have this:

```
        config SAMPLE_KEYPHRASE
»       »       depends on CAVS || IMX                                                                                                                                                                     
»               bool "Keyphrase test com
```

the CONFIG_SAMPLE_KEYPHRASE will be automatically selected.